src/CheckoutSdk/Disputes/GetDisputesRequest.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using System.Net;
+
+namespace Checkout.Disputes
+{
+    /// <summary>
+    /// Defines a request for a get disputes request.
+    /// </summary>
+    public class GetDisputesRequest
+    {
+        /// <summary>
+        /// Creates a new get disputes request.
+        /// </summary>
+        /// <param name="limit">The numbers of results to return. 1 .. 250, default is 50.</param>
+        /// <param name="skip">The number of results to skip, default is 0.</param>
+        /// <param name="from">The ISO-8601 date and time from which to filter disputes, based on the dispute's last_update field.</param>
+        /// <param name="to">The ISO-8601 date and time until which to filter disputes, based on the dispute's last_update field.</param>
+        /// <param name="id">The unique identifier of the dispute.</param>
+        /// <param name="statuses">One or more comma-separated statuses. This works like a logical OR operator.</param>
+        /// <param name="paymentId">The unique identifier of the payment.</param>
+        /// <param name="paymentReference">An optional reference (such as an order ID) that you can use later to identify the payment.</param>
+        /// <param name="paymentArn">The acquirer reference number (ARN) that you can use to query the issuing bank.</param>
+        public GetDisputesRequest(
+            int? limit = null,
+            int? skip = null,
+            string from = "",
+            string to = "",
+            string id = "",
+            string statuses = "",
```

Comment:
There's no need to initialise the string params as an empty string - you can leave these as `null`